src/main/java/Duke.java

```diff
@@ -1,10 +1,126 @@
+import java.util.Scanner;
+
 public class Duke {
+    static final String LINE = "____________________________________________________________";
+
```

Comment:
I think it's better to put the main method at the bottom and declare other methods at first, which is more comfortable to read that people don't need to scroll down the code frequently to figure out where you declare and how you implement it.